frontend/src/components/toolbox/AppBar.tsx

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import styled from 'styled-components';
+
+export const AppBar = ({homeHref, aboutHref, homeTitle, aboutTitle} : any) => {
+  return (
+  <StyledDiv>
+
+    <StyledSection>
+      <StyledA href={homeHref}>{homeTitle}</StyledA>
+    </StyledSection>
+
+    <StyledSection>
+      <span>|</span>
+    </StyledSection>
+
+    <StyledSection>
+      <StyledA href={aboutHref}>{aboutTitle}</StyledA>
+    </StyledSection>
+
+  </StyledDiv>
+  );
+};
+
+const StyledSection = styled.section`
```

Comment:
Try and refactor the styled components into a separate file, for example `AppBar.theme.ts`
Then import it using 
`
import * as s from './AppBar.theme';
`
And from then on, refer to the styled components as `s.StyledSection` (etc)
This will help us keep track of what's a styled component and what's not.